src/screens/novel/components/LoadingAnimation/NovelScreenLoading.tsx

```diff
@@ -68,7 +68,7 @@ const NovelInformation = memo(() => (
   </View>
 ));
 
-const ChapterItem = memo(({ index }: { index: number }) => (
+const ChapterItem = memo(() => (
```

Comment:
lint was mad at unused variable so i removed it

Reply:
maybe lint gives problems because Traitor latest commit bypassed lint verification.